openshift4/templates/backend.dc.yaml

```diff
@@ -315,6 +315,16 @@ objects:
                     secretKeyRef:
                       key: common-services-email-host
                       name: common-services
+                - name: EMAIL_ENABLED
```

Comment:
Not sure if we need a secret for a bool flag

Reply:
No, but it's nicer to control it via OpenShift rather than having to make a code change and redeploy to change the value?

Reply:
Fair, the current setup is a clunky developer experience.